wire-ios/Wire-iOS/Sources/UserInterface/Settings/DeviceManagement/ClientListViewController.swift

```diff
@@ -178,7 +178,7 @@ final class ClientListViewController: UIViewController,
         super.viewWillAppear(animated)
         self.clientsTableView?.reloadData()
         self.navigationController?.setNavigationBarHidden(false, animated: false)
-        setupNavigationBarTitle(L10n.Localizable.Registration.Devices.title.capitalized)
```

Comment:
How come we no longer want these titles to be capitalized? Is it because this should be defined at the localization level?

Reply:
> Is it because this should be defined at the localization level?

👍